challenges/extreme-constructor-parameter/question.py

```diff
@@ -0,0 +1,32 @@
+"""
+TODO:
+
+Define a decorator `constructor_parameter` that input target class 
+and return a wrapper function with the same signature as the constructor of the target class, 
+and function which decorated by `constructor_parameter` can be called with the instance of the target class.
```

Comment:
remove "which"

Reply:
Let's be specific:

function decorated by `constructor_parameter` can be called with an instance `Foo`